### DISCUSSION TITLE
ClawPDF Network Printer Works with Administrator but Not with User Account

### DISCUSSION THREAD

**Discussion by register0101** [clawsoftware/clawPDF#153] [General]
Hello,

I'm facing an issue with setting up ClawPDF as a network printer in our environment. Here's the situation:

Setup Details:

Operating System: Windows Server Standard 2022
ClawPDF Version: [0.9.3]
Client Machines: Windows 11, domain-joined
User Account: clawpdf (a non-administrator domain user)
Domain Name: test
Administrator Account: test\Administrator
The Problem:

When I enable the network printer functionality using the Administrator account with the following command, everything works perfectly, and clients can print without any issues:

& "C:\Program Files (x86)\clawpdf\clawPDF.Bridge.exe" /networkprinter=enable /username=Administrator /domain=test

However, when I try to enable the same network printer functionality using a non-administrator domain account (clawpdf), clients are unable to print. The command used is:

& "C:\Program Files (x86)\clawpdf\clawPDF.Bridge.exe" /networkprinter=enable /username=clawpdf /domain=test

What I've Tried:

The clawpdf user has been granted Full Control permissions on the folder where ClawPDF saves PDFs.

The clawpdf user has been given Print and Manage Documents permissions on the ClawPDF printer.

I've confirmed that the clawpdf user is part of the Domain Users group.

Firewall settings i have switch off 

Tested using a local service account as well, but the issue persists when not using the administrator account.
Symptoms:

When using the clawpdf account, clients see the ClawPDF network printer, but print jobs either do not get processed or are not saved in the designated folder.

No obvious errors appear in the Event Viewer or ClawPDF logs related to this issue.

Request for Help:

Has anyone encountered a similar issue where ClawPDF works with an administrator account but not with a non-administrator user account?

Are there any specific permissions or configurations that I might be missing to get this working with a standard user account?

Any suggestions on how to debug this further would be greatly appreciated.

Thank you in advance for your help!

**Comment by masc-mrng** [comment]
Hi, 

I don't know if this information helps:
I did some tests in a virtual environment and the network printing seemed not to work in the beginning. 
How does my setup look like?

A COM+ application, which is running with the credentials of a "service-user" (not GMSA -> local windows account which is denied to log on locally by GPO) triggers the print. This "service-user" has local admin rights due to requirements of the COM+ Component. 
The printer (clawPDF) was installed using the same credentials of this "service-user". 

The network printing functionality did not work - no matter what I configured. 
Then I changed the credentials for the COM+ Application as well as for clawPDF to the "admin-user" which has also local admin rights and I was logged on with this user. After this change printing via network worked BUT only as long as I was logged on to the server with the "admin-account". 

So I guess this might be the issue in your case too, as a standard user normally not logs on to the server? 

Cheers
Martin
